mlflow/ml-package-versions.yml

```diff
@@ -879,3 +879,15 @@ promptflow:
       "< 1.18": ["numpy<2"]
     run: |
       pytest tests/promptflow/test_promptflow_model_export.py
+
+litellm:
+  package_info:
+    pip_release: "litellm"
+    install_dev: |
+      pip install git+https://github.com/BerriAI/litellm.git
+  autologging:
+    minimum: "1.52.0"
```

Comment:
self-TODO: Update minimum version once the callback PR in LiteLLM side is merged.